pkgs/by-name/st/stats/package.nix

```diff
@@ -25,12 +30,23 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
+  passthru.updateScript = lib.getExe (writeShellApplication {
+    name = "stats-update-script";
+    runtimeInputs = [ curl jq common-updater-scripts ];
+    text = ''
+      set -euo pipefail
+      url="$(curl --silent "https://api.github.com/repos/exelban/stats/tags?per_page=1")"
+      version=$(echo "$url" | jq -r '.[0].name' | cut -c 2-)
```

Comment:
probbaly want to double quote this one too

Reply:
Done